src/commands/node.mjs

```diff
@@ -2627,21 +2598,61 @@ export class NodeCommand extends BaseCommand {
           self.logger.info(`nodeId: ${nodeId}`)
           self.logger.info(`config.newAccountNumber: ${config.newAccountNumber}`)
 
-          self.logger.info(`ctx.signingCertDer: ${ctx.signingCertDer}`)
-          self.logger.info(`ctx.tlsCertHash: ${ctx.tlsCertHash}`)
           try {
             const nodeUpdateTx = await new NodeUpdateTransaction()
               .setNodeId(nodeId)
 
-            if (config.newGRPCHash) {
-              const tlsCertDer = await this.loadPermCertificate(config.newCACertificate)
+            if (config.tlsKey) {
+              const tlsCertDer = await this.loadPermCertificate(config.gossipKey)
               const tlsCertHash = crypto.createHash('sha384').update(tlsCertDer).digest()
```

Comment:
I think the hash is supposed to be of the public key, right?

Reply:
I think you need to generate the public key from the private key, then take the hash of the public key.

Reply:
Updated